packages/scxa-cell-type-wheel-experiment-heatmap/src/CellTypeWheelExperimentHeatmap.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import _ from 'lodash'
+
+import MarkerGeneHeatmap from '@ebi-gene-expression-group/scxa-marker-gene-heatmap/lib/MarkerGeneHeatmap'
+import CellTypeWheel from '@ebi-gene-expression-group/scxa-cell-type-wheel'
+
+class CellTypeWheelExperimentHeatmap extends React.Component {
+  constructor(props) {
+    super(props)
+  }
+
+  render() {
+    const { cellTypeWheelData, cellTypeWheelSearchTerm, cellTypeHeatmapSearchTerm, cellTypeHeatmapData } = this.props
```

Comment:
`cellTypeHeatmapSearchTerm` and `cellTypeHeatmapData` are not being passed in props. Please add these two in props as well.